packages/svelte/src/compiler/phases/3-transform/client/visitors/CallExpression.js

```diff
@@ -82,7 +82,9 @@ export function CallExpression(node, context) {
 		['debug', 'dir', 'error', 'group', 'groupCollapsed', 'info', 'log', 'trace', 'warn'].includes(
 			node.callee.property.name
 		) &&
-		node.arguments.some((arg) => arg.type !== 'Literal') // TODO more cases?
+		node.arguments.some(
+			(arg) => arg.type === 'SpreadElement' || context.state.scope.evaluate(arg).values.has(UNKNOWN)
```

Comment:
Evaluation of SpreadElement will result in UNKNOWN anyway, so the first check can be removed, right?
And maybe it's cleaner to add `has`/`is_unknown` prop-flag to `Evaluation` and use it here.